fv3core/fv3core/initialization/baroclinic.py

```diff
@@ -423,63 +425,65 @@ def init_baroclinic_state(
     Create a DycoreState object with quantities initialized to the Jablonowski &
     Williamson baroclinic test case perturbation applied to the cubed sphere grid.
     """
-
-    state = DycoreState.init_empty(metric_terms.quantity_factory)
-    nx, ny, nz = local_compute_size(state.delp.data.shape)
+    sample_quantity = metric_terms.lat
```

Comment:
can we extract the initialization of the numpy state out in it's own function?

Reply:
yes! good idea